### ISSUE TITLE
TypeError: Cannot read property 'array' of undefined with @nativescript/android 8.2.4

### ISSUE THREAD

**Issue by sebj54** [nativescript-community/arraybuffers#1]
```
TypeError: Cannot read property 'array' of undefined
    at pointsFromBuffer (file:///data/data/com.my-app/files/app/vendor.js:2:17268)
    at arrayToNativeArray (file:///data/data/com.my-app/files/app/vendor.js:2:17390)
    at Bluetooth.writeValueToChar (file:///data/data/com.my-app/files/app/vendor.js:2:48200)
    at file:///data/data/com.my-app/files/app/vendor.js:2:49827
    at Bluetooth.attachSubDelegate (file:///data/data/com.my-app/files/app/vendor.js:2:61827)
    at file:///data/data/com.my-app/files/app/vendor.js:2:49384
    at new Promise (<anonymous>)
    at file:///data/data/com.my-app/files/app/vendor.js:2:48828
    at file:///data/data/com.my-app/files/app/vendor.js:2:44420
    at async file:///data/data/com.my-app/files/app/vendor.js:2:10817
```

### Which platform(s) does your issue occur on?
- Android
- All versions?
- device: Xiaomi A1

### Please, provide the following version numbers that your issue occurs with:

- CLI: 8.2.3
- Cross-platform modules: 8.2.5
- Android Runtime: 8.2.4
- Plugin(s): many (not relevant)

### Please, tell us how to recreate the issue in as much detail as possible. 

This issue is caused by this one: https://github.com/NativeScript/android-runtime/issues/1715

It is thrown **only in release builds**, it works fine with debug builds. The issue appears when I try to write a characteristic with the BLE plugin.

I don't know how this issue should be fixed, but I wanted you to know it exists. Maybe solving the issue in @nativescript/android will be enough, but maybe the version check should be made with another function or variable?

**Comment by facetious** [comment]
This is resolved in [android-runtime@8.3](https://github.com/NativeScript/android-runtime/releases/tag/v8.3.0).
> fix: Set package version before preReleaseBuild by @sebj54 in https://github.com/NativeScript/android-runtime/pull/1716

**Comment by sebj54** [comment]
Seems I figured out how to fix the issue 😅 Thanks for the heads up!
